packages/dd-trace/src/exporters/agent/index.js

```diff
@@ -7,15 +7,15 @@ const Writer = require('./writer')
 class AgentExporter {
   constructor (config, prioritySampler) {
     this._config = config
-    const { url, hostname, port, lookup, protocolVersion, stats = {} } = config
+    const { url, hostname, port, lookup, protocolVersion, stats = {}, appsec } = config
     this._url = url || new URL(format({
       protocol: 'http:',
       hostname: hostname || 'localhost',
       port
     }))
 
     const headers = {}
-    if (stats.enabled) {
+    if (stats.enabled || appsec?.standalone?.enabled) {
```

Comment:
In `packages/dd-trace/src/span_stats.js`, to enable stats, you are checking `enabled && !appsec?.standalone?.enabled` but here, you are doing different check, `||` instead of `&& ! `.

Are both ok? It seems like both should be the same check.

Reply:
stats must be disabled if `appsec?.standalone?.enabled`

and `'Datadog-Client-Computed-Stats': 'yes'` must be sent if `appsec?.standalone?.enabled`

Reply:
ok, understood. we want to send 'Datadog-Client-Computed-Stats': 'yes'` even when stats are disabled if we are in standalone mode. 
Understood 👍